addons/web/static/tests/views/list_view_tests.js

```diff
@@ -101,7 +101,11 @@ QUnit.module("Views", (hooks) => {
                         o2m: { string: "O2M field", type: "one2many", relation: "bar" },
                         m2m: { string: "M2M field", type: "many2many", relation: "bar" },
                         amount: { string: "Monetary field", type: "monetary" },
-                        amount_currency: { string: "Monetary field (currency)", type: "monetary", currency_field: "company_currency_id" },
+                        amount_currency: {
+                            string: "Monetary field (currency)",
+                            type: "monetary",
+                            currency_field: "company_currency_id",
+                        },
```

Comment:
This diff is a result of "formatting" the file. Is it okay to keep this diff? Same is true for "currency_field is taken into account when formatting monetary values" below.